inc_internal/internal_model.h

```diff
@@ -21,6 +21,15 @@ limitations under the License.
 #include "ziti/model_support.h"
 #include "ziti/ziti_model.h"
 
+#define ZITI_IDENTITY_DATA_MODEL(XX, ...) \
```

Comment:
I'm curious why you added a new model instead of adding to the existing identity model?